static/src/javascripts/lib/defer-to-analytics.js

```diff
@@ -6,7 +7,7 @@ mediator.on('analytics:ready', () => {
     analyticsReady = true;
 });
 
-function deferToAnalytics(afterAnalytics) {
+function deferToAnalytics(afterAnalytics: () => void) {
```

Comment:
I'm not sure how far we're intending to go with the Flow, but should we specify a return type of `void` for the `deferToAnalytics` function itself?